doc/explanation/microcloud.rst

```diff
@@ -26,8 +26,30 @@ Networking
 
 By default, MicroCloud uses MicroOVN for networking, which is a minimal wrapper around OVN (Open Virtual Network).
 
-MicroOVN requires an uplink network that is an actual L2 subnet (which is usually not the case in a virtual cloud environment).
-In addition, MicroOVN requires its own dedicated network interface, for example, a dedicated physical network interface, a VLAN, or a virtual function on an :abbr:`SR-IOV (Single root I/O virtualisation)`-capable network interface.
+For outside connectivity, MicroOVN requires an uplink network.
+This uplink network must be an actual L2 subnet (which is usually not the case in a virtual cloud environment).
+
+Each machine in the MicroCloud cluster must have two available network interfaces:
+
+.. _microcloud-networking-intracluster:
+
+Network interface for intra-cluster traffic
+  MicroCloud requires one network interface that is configured with an IP address of a real layer 2 network.
+  All cluster members must be connected to the same network.
+
+  This network interface can be, for example, a dedicated physical network interface, a VLAN, or a virtual function on an :abbr:`SR-IOV (Single root I/O virtualisation)`-capable network interface.
+  It serves as the dedicated network interface for MicroOVN and is used for multicast discovery (during setup) and all internal traffic between the MicroCloud, OVN, and Ceph members.
+
+.. _microcloud-networking-uplink:
+
+Network interface to connect to the uplink network
+  MicroCloud requires one network interface for connecting OVN to the uplink network.
+  This network interface must either be an unused interface that does not have an IP address configured, or a bridge.
+
+  MicroCloud configures this interface as an uplink interface that provides outside connectivity to the MicroCloud cluster.
+
+  You can use different interfaces on each cluster member.
+  MicroCloud will connect all interfaces to the same uplink network, using the gateway and IP address range information that you provide during the MicroCloud initialisation process.
```

Comment:
I don't quite understand this bit:

> MicroCloud will connect all interfaces to the same uplink network...

What is "interfaces" here?

I think this should say something along the lines of:

> MicroCloud will configure the distributed OVN network(s) to use the same uplink network for external connectivity...

Reply:
OK how about this:

> You can specify a different interface to be used for the uplink for each cluster member.
> MicroCloud will requires that all interfaces are connected to the uplink network, using the gateway and IP address range information that you provide during the MicroCloud initialisation process.

Reply:
Thanks, updated. :)